docs/_includes/common/tooltip.html

```diff
@@ -110,9 +112,70 @@ <h3 class="infotip__heading">Infotip</h3>
             </div>
         </div>
     </div>
+    </span>
```

Comment:
Is this an errant `</span>`? It popped up in the most recent change, but without a corresponding opening tag.

Reply:
Good catch, was an issue with copy and pasting 👍